assets/js/collaborative-editor/components/inspector/JobForm.tsx

```diff
@@ -105,24 +104,14 @@ export function JobForm({ job }: JobFormProps) {
     `jobs.${job.id}` // Server validation automatically filtered to this job
   );
 
-  // Y.Doc sync
-  useWatchFields(
-    job,
-    changedFields => {
-      Object.entries(changedFields).forEach(([key, value]) => {
-        if (key in form.state.values) {
-          if (key === 'adaptor' && value) {
-            const { package: adaptorPackage } = resolveAdaptor(value);
-            if (adaptorPackage) {
-              form.setFieldValue('adaptor_package', adaptorPackage);
-            }
-          }
```

Comment:
Is this removal intentional? If `adaptor_package` needs to be derived from `adaptor` when syncing from Y.Doc, couldn't this lead to a regression ?

Reply:
I've confirmed that both `adaptor` & `adaptor_package` values come through ydoc. Hence, we might not need to resolve whenever we get an update. 

We already handle such resolution in the `handleAdaptorSelect` function in this file